src/utils/teamScraper.py

```diff
@@ -0,0 +1,128 @@
+import json
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+database_id = os.environ.get('NOTION_DB_ID')
+url = f"https://api.notion.com/v1/databases/{database_id}/query"
+headers = {
+    "accept": "application/json",
+    "Notion-Version": "2022-06-28",
+    "Authorization": os.environ.get('NOTION_TOKEN')
+}
+query = {
+    "filter": {
+        "property": "Status",
+        "select": {
+            "equals": "Current Member"
+        }
+    }
+}
+
+eboard_roles = set(["Executive Director", "E-Board Liason", "Technical Director",
+                    "UX Director", "Marketing Director", "Operations Director"])
+head_ofs = set(["Head of Recruiting", "Head of UX",
+               "Head of Project Acquisition", "Head-of Community", "Head of DX"])
+# output_file = "src/content/team/team.json"
+output_file = "new_team.json"
+
+
+def get_current_members_from_notion():
+    response = requests.post(url, headers=headers, json=query)
+    data = response.json()
+    results = data["results"]
+    return results
+
+
+def make_person(name, team_name, role, linkedin, portfolio, email):
+    social_media = {
+        "email": email or None,
+        "linkedIn": linkedin or None,
+        "portfolio": portfolio or None
+    }
+    social_media_filtered = {k: v for k,
+                             v in social_media.items() if v is not None}
+
+    return {
+        "name": name,
+        "team": {
+            "name": team_name,
+            "role": role,
+        },
+        # "profileImage": "" if email is None else './profileImages/' + ''.join([name.capitalize() for name in email[0: email.index('@')].split('.')]) + ".jpg",
```

Comment:
If we can use Pillow to get profile image from notion, script can then save it to a file locally, naming convention in the PR summary above